### DISCUSSION TITLE
Conversational Actions have been sunset

### DISCUSSION THREAD

**Discussion by CasahDev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#903] [Q&A]
Hi, i just tried to create a project on the actions system of google and i saw that i couldn't register any model due to google's update of June 13th. Do anyone can tell me how to bypass this and makes me use this project ?
![image](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/assets/37735933/c3b7e5d0-f70d-4427-a8e5-66f3edda99a0)
I really appreciate the effort you would make to help me.

**Comment by im-coder-lg** [comment]
I don't think anyone can bypass that. Seems like Google really wants to stop people using the GA API to make apps. Man, I miss the old days!
